<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TiltAngleGroupBox</class>
 <widget class="QGroupBox" name="TiltAngleGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>GroupBox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="startIncrementBtn">
     <property name="text">
      <string>Specify the Start angle and step (degrees)</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="startLabel">
       <property name="text">
        <string>Starting Angle:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="startingAngle">
       <property name="text">
        <string>-60</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="incrementLabel">
       <property name="text">
        <string>Increment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="increment">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="rawTltBtn">
     <property name="text">
      <string>Tilt Angles in existing rawtlt file</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLineEdit" name="rawTltFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectRawTlt">
       <property name="text">
        <string>Select...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>398</width>
       <height>118</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
